### ISSUE TITLE
cant seem to find teh precicion bow , in the fabricator bench, probbely overlooking something again 

### ISSUE THREAD

**Issue by RaShCl** [Jimk72/Icarus_Mods#35]
Untill now all your mods  seem to work for me, except the precision bow.
I am looking in teh fabricator bench, but when i click the bow tab, or looking over all the items in that bench, i still cant find it.
I did install it thrue icarus mod manager, it sais its installed.
I also looked  in your workshop tab on teh station, maybe it was something i needed to get from there
So i was wondering is it outdated for this latest version of icarus ?
Or again do i forget  to do something ?

here is a link to teh screenshot of teh mod manager, maybe i am overlooking something :P

thx in advance and let teh mods coming :P

**Comment by Jimk72** [comment]
I haven't heard of any other issues. I'm currently working 13hr shifts(turnaround at work) but next time I get a chance I'll load it up and see if there is an issue!

**Comment by RaShCl** [comment]
np take your time, rl always comes first i am not running away screaming yet :P

**Comment by Jimk72** [comment]
Had a chance to test it tonight and there were no issues. I noticed you are using a very old version of the mod manager. Current version is 2.2.2. It is not compatible with the version you have installed so you would have to delete that install and download the latest version.  From what I see in your image I believe all those mods are updated and avail so you should be able to just re download them in IMM downloads page.

**Comment by Jimk72** [comment]
The bow should be the last item in the fabricator bench as custom items are added to the end of the recipes list.

**Comment by RaShCl** [comment]
hmm weird i click update mods, so the mod manager itself doesn't update ?

So I need to download a new mod manager every x days, and install it?

maybe the developers of the mod manager should put an auto-updater in their manager
but i will download a new version then thx for the help

**Comment by RaShCl** [comment]
oki i did a compleet new install of teh latest mod manager and all the mods, following your youtube  tutorial
as you can see  i have the latest version and all the mods including the  precision bow are in the extracted mod mods folder.

https://gyazo.com/e155f66743135ac2766a692550d53bfa

But still no bow in the  machining bench after loading in my game ??

all other mods are in gamer alive and working.

**Comment by Jimk72** [comment]
Should be fabricator bench, not the machining bench.

**Comment by RaShCl** [comment]
sorry thats what i ment, i always confuse those 2

looking at teh fabricator right now and no precician bow anywhere

**Comment by RaShCl** [comment]
will try to install only your bow mod ,then we can see if its a mod conflict, :P 
will let you know later on , again thx

**Comment by Jimk72** [comment]
Ok let me know.

**Comment by RaShCl** [comment]
> Ok let me know.
after only running teh precision bow mod still nothing.

Then i decided to reinstall Icarus the mod manager and the mod completely fresh, running it with as few programs possible, like virus scanners and stuff like that in the background.

Then I made a new character and started a new game still no bow  for some reason,

Well it is what it is , there can be so many different conflicts on pc's with so many configurations,
I  think at some point it will be fixed, as i said its not that big of a problem.

All your other mods i use , are working, and i use them more then teh bow :P

and a auto-updater to Mod manager would be cool 📦 

Thx for all teh fast responses, and as i said before keep teh mods coming

ofcourse i will keep tinkering and see what i can do more to have your mod working

**Comment by Jimk72** [comment]
There is a workshop version of that bow also. Maybe try that. As for auto updater there is a button check for IMM update that will auto download and update the mod manager if there is an update avail. I didn't want to do an auto check when ran as I want people to know when IMM is connecting to the web. Just for incase they download a hacked version of imm from some one else they will know it should not try to connect unless you tell it to. Update mods database, download mods, download unrealpak and check for IMM update should be the only time IMM connects to web.

**Comment by RaShCl** [comment]
> There is a workshop version of that bow also. Maybe try that. As for auto updater there is a button check for IMM update that will auto download and update the mod manager if there is an update avail. I didn't want to do an auto check when ran as I want people to know when IMM is connecting to the web. Just for incase they download a hacked version of imm from some one else they will know it should not try to connect unless you tell it to. Update mods database, download mods, download unrealpak and check for IMM update should be the only time IMM connects to web.

ahh oki i will try that one ,
And i can see why you dont want a auto updater, its easy these days to have a  fake   copy of a program
and lure people in didn't think about that.

i will try and see if i see teh workshop version of teh bow in the workshop will let you know ofcourse as i said  on 1 occasion teh kiwi fruit as gone from the workshop ,but later returned  from itself.

1 more thing do you think its possible to have the teleporter have multiple  spawn points , i mean more then two ?
would be cool to finally have a tp  to all important stuff i have placed around teh map 

like you activate a tp and it asks  where you ant to tp to, like a small list ?

anyway thx for all teh fast responses and help with the bow , and cant wait to see more mods, i like them a lot

have a good day/night

john

**Comment by RaShCl** [comment]
I did check your stuff in teh workshop and only
2 items i found there.

Crate Full of LED lights and a Teleporter kit

**Comment by Jimk72** [comment]
So It's been bothering me that all the other mods work except that one so looking through it I realized I never asked you if you had the compound bow unlocked! It requires the compound bow to be unlocked before you can craft the Precision bow. Can you check if you unlocked that?

**Comment by RaShCl** [comment]
Yes, that was the problem, maybe put it in teh mod info tab in the mod manager. :)

I do admit it's a nice bow It's a nice touch having a range scope on the precision bow.

But you need to wait till tier 4, at which time most people have enough money to buy a better bow in teh workshop.
also when people are tier 4, they mostly have a good enough weapon to harvest animals for meat.

if think it would be better, if it was put in the end tier 2 and begin tier 3
